zips/zip-0226.rst

```diff
@@ -139,6 +139,8 @@ It consists of
 
 .. math:: (\mathsf{leadByte} : \mathbb{B}^{\mathbb{Y}}, \mathsf{d} : \mathbb{B}^{[\ell_{\mathsf{d}}]}, \mathsf{v} : \{0 .. 2^{\ell_{\mathsf{value}}} - 1\}, \mathsf{rseed} : \mathbb{B}^{\mathbb{Y}[32]}, \mathsf{asset\_base} : \mathbb{B}^{[\ell_{\mathbb{P}}]}, \mathsf{memo} : \mathbb{B}^{\mathbb{Y}[512]})
 
+We make explicit the order of addition of the note commitments to the note commitment tree in ZIP 227 [#zip-0227-note-commitment-order]_.
```

Comment:
Instead of repeating the `We make stuff..` wording, prefer passive `The explisit order of additions is specified in ...`

Reply:
Done